spec/apps/invalid-unload/invalid-unload.spec.js

```diff
@@ -20,31 +20,21 @@ describe(`invalid-unload app :`, () => {
     errs = [];
     singleSpa.addErrorHandler(handleError);
 
-    return import('./invalid-unload.app.js')
-      .then(app => myApp = app)
-      .then(() => singleSpa.unloadApplication('./invalid-unload.app.js'))
```

Comment:
this promise would sometimes never resolve nor reject :'( which would cause test failures.

We should maybe look into that further at some point, but after 5+ hours debugging I decided to punt on it.